### PR TITLE
{chem}[GCCcore/8.2.0] Molden v6.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2496.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0593971faa6f980054caf30b83a02706 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3164.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/25123f3ab5051dc5bb909405cd5bceac for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2058.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c9de67e676bf7eca7ef3dc77a1ae32c9 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5606ee68c47ca36ec3468f090abb5d58 for a full test report.

File: easybuild/easyconfigs/m/Molden/Molden-6.1-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+name = 'Molden'
+version = '6.1'
+
+homepage = 'http://www.cmbi.ru.nl/molden/'
+description = """Molden is a package for displaying Molecular Density from the
+ Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical
+ packages Mopac/Ampac"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```
Comment (Micket) [inline easybuild/easyconfigs/m/Molden/Molden-6.1-foss-2019a.eb]: This looks like it can definitely be dropped all the way down to GCCcore.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**